pickup/urls.py

```diff
@@ -4,6 +4,8 @@
 
 urlpatterns = [
     path('', views.index, name='index'),
+    path('home/', views.home, name='home'),
+    path('profile/', views.profile_list, name='Profile List'),
```

Comment:
This line (8) should be removed. That view is no longer used. There is another profile view that is below.

Reply:
Thank you so much!